workflows/qc/inputs/multiqc_config.yaml

```diff
@@ -0,0 +1,53 @@
+extra_fn_clean_exts:
+    - '.ValidateSamFile'
+    - '.whole_genome'
+    - '.exon'
+    - '.CDS'
+    - '.MarkDuplicates'
+log_filesize_limit: 1000000000
+fastqc_config:
+  fastqc_theoretical_gc: "hg38_genome"
```

Comment:
So we're assuming that we're always using GRCh38. It'd be nice to make this configurable, but I don't know if there is a better way to do that. Maybe we just name the file `multiqc_config_hg38.yaml` and then we could have preconfigured choices for appropriate genomes. I wouldn't actually create the others at this point.

Reply:
I like renaming this and hosting multiple in the future. We're going to want target specific configs in the near future (me and David have discussed differentially altering how the coverage sections appear for our 3 targets (WGS/WES/RNA)).

Biggest problem with this approach is that the number of configs here might get out of hand. I'd say we can plan to host `multiqc_config_[WGS,WXS,RNA-Seq]_hg38.yaml` in the near future. I think hosting 3 configs is reasonable. But do we really want that number increasing further down the line?

I haven't actually tried visualizing it yet, but in the docs there's `fastqc_theoretical_gc: "hg38_transcriptome"` (that might not be the correct keyword). Not sure what it looks like but it's been on my agenda to check out. That could present another config option to host. Then we could be looking at hosting 4 nearly identical configs.

I think statically hosting these is fine for now. But it's worth considering for the future what our line will be. At what point do we stop putting these different configs up?

Reply:
I think we would host any configs that we're using. Beyond that, unless there is some glaring omission that would be generally relevant, I wouldn't host others.